security/security.md

```diff
@@ -1,18 +1,137 @@
+
+It is also possible to have automatic backups create bad backups when, for example, some change or hack has happened to a WordPress website that goes undetected for long enough for the automatic backups to run. Depending on how long it takes to identify that a hack has occurred or there is some problem with WordPress, the automatic backup plugin may have enough time to overwrite any good backups that had been made before the hack or problem occurred.
+
+Automatic backups can also push your hosting account or server's disk space beyond the limits your hosting provider has for automatically backing up your hosting account or server on their system, if they offer automated backups.
+
+Some hosting providers also do not allow you to keep backups on your hosting account or server and may delete the backups without your consent after a certain period of time.
+
+**Consider reviewing your hosting provider's policies on backups and the storage of backups. Many hosting providers can provide good advice and support for setting up thorough, multi-level backup solutions for WordPress.**
+
+### Manually Backing Up WordPress
+You are not required to use a WordPress backup plugin to backup WordPress. The WordPress files and database can be manually copied and exported using a variety of different tools and techniques depending on the levels of access you have to your hosting account's files and databases.
+
+#### Use Hosting Provider's Control Panel or Tools
+Most hosting providers use control panels or administrative tools that let you create different kinds of backups through their interfaces. Consult your hosting provider's documentation or contact their support team for assistance using their tools to generate backups of WordPress.
+
+#### Backing Up WordPress Files
+WordPress files can be backed up using any tool that allows you to access and to download files from your hosting account. These include but are not limited to FTP clients, command-line/shell clients, and web clients. Consult your hosting provider's documentation or contact their support team for assistance with using FTP, command-line, or web clients to access and to download your files to your computer.
+
+#### Backing Up WordPress Database
+The WordPress database is not stored in a file in your hosting account that can be easily or simply downloaded or copied. It must be exported using a database backup tool. Most hosting providers offer tools for accessing and backing up databases. If your hosting account does not have a database management tool, you can install phpMyAdmin, a popular, open-source tool for managing MySQL databases (the WordPress database is usually a MySQL database). [phpMyAdmin's makers have written documentation for installing phpMyAdmin](https://docs.phpmyadmin.net/en/latest/setup.html). After installing phpMyAdmin, you should be able to use your WordPress database username and database password from your wp-config.php file to login using phpMyAdmin.
```

Comment:
#### Security of the configuration file
The wp-config.php file contains credentials for database and so on. After WordPress installation, the wp-config.php file can be executed. If, for some reason, processing of PHP files by the web server is turned off, hackers can access the content of the wp-config.php file. Hosters should verify that unauthorized access to the wp-config.php file is blocked.